### PR TITLE
Teamanalytics

### PR DESCRIPTION
**Pull request by nikunj1705**

### CODE REVIEW

File: docs/agent-chat/analytics.md [hunk 1]
```diff
@@ -25,11 +25,11 @@ title: Analytics
 
 1. **Total Chats Received** 
     
-    All ongoing chats that started in the chosen time duration. This includes chats in waiting, queued chats as well as completed chats. The total of all chats in the selected time duration from the time-date picker.
+    All ongoing chats for a chosen Team view, that started in the selected time duration. This includes chats in waiting, queued chats and pinned chats for the Team in question. The total number of all chats received i.e. `first_message_time` in the selected time duration from the time-date picker.
```
Comment (GanadiniAkshay) [inline docs/agent-chat/analytics.md:28]: Please change it to Team View

Comment (RandiveSushant) [inline docs/agent-chat/analytics.md:28]: So, should we mention here a note saying that these are chats which are **picked up by an agent** which means.. for all those chats which were marked complete when the team was offline _(and thus never went to the team and therefore any agent)_ are not counted in this.

File: docs/agent-chat/index.md [hunk 2]
```diff
@@ -2,11 +2,11 @@
...
 ---
 
-Chat Solutions at Haptik can be powered entirely by Artificial Intelligence (AI) or by a combination of AI and Chat Agents. We have built a *Live Chat* tool for it. This tool is created for operationally managing user chats.
+Chat Solutions at Haptik can be powered entirely by Artificial Intelligence (AI) or by a combination of AI and Chat Agents. We have built a Live *Agent Chat* tool for it. This tool is created for operationally managing user chats.
 
-## Using *Live Chat*
+## Using Live *Agent Chat*
 
-To use this tool, users are to be created with a specific set of permissions. The users can then start monitoring and taking daily chats on their business. *Live Chat* tool has the same capabilities of the AI along with additional functionalities such as chat flow analytics, assignment of chats by managers etc.
+To use this tool, users are to be created with a specific set of permissions. The users can then start monitoring and taking daily chats on their business. Live *Agent Chat* tool has the same capabilities of the AI along with additional functionalities such as chat flow analytics, assignment of chats by managers etc.
```
Comment (RandiveSushant) [inline docs/agent-chat/index.md:9]: when you say **Live *Agent Chat* tool has the same capabilities of the AI** do you mean to say **Live *Agent Chat* tool has the same UI capabilities of the AI bot**?

File: docs/agent-chat/analytics.md (hunk 1 above)
Comment (nikunj1705) [reply docs/agent-chat/analytics.md:28]: Good one. Adding.

File: docs/agent-chat/index.md (hunk 2 above)
Comment (nikunj1705) [reply docs/agent-chat/index.md:9]: Yeah.

**Pull request merged**